ekf/meas.c

```diff
@@ -39,11 +40,11 @@
 #define BARO_CALIB_AVG 100
 
 static struct {
-	kalman_calib_t calib;
+	meas_calib_t calib;
 } meas_common;
 
 /* accelerometer calibration data */
-float acc_calib1[12] = {
+static float acc_calib1[12] = {
```

Comment:
In the name of array, there is a magic number. Can you change it ?
```suggestion
static const float accCalib1[12] = {
```

Reply:
Used two macros: 
```c
/* different accelerometer biases sizes */
#define ACC_BIAS_SIZE    3
#define ACC_NONORTHO_SIZE 9
```
Precompiled array will be dropped as soon as config parser will be used